lib/main_common.pm

```diff
@@ -2434,9 +2434,37 @@ sub load_host_installation_modules {
     loadtest "installation/reboot_after_installation";
 }
 
+sub set_mu_virt_vars {
+    # Set UPDATE_PACKAGE based on BUILD(format example, BUILD=:33310:dtb-armv7l)
+    my $BUILD = get_var('BUILD', '');
```

Comment:
get_required_var ?

Reply:
Thanks for the review. Yes, agree with you. Given that A5 task is coded on top of this commit, will update this later in another PR.